reg_tests/rt.sh

```diff
@@ -16,8 +16,10 @@ cd ${WORK_DIR}
 rm -f reg_test_results.txt
 rm -rf UFS_UTILS
 
-git clone --recursive https://github.com/ufs-community/UFS_UTILS.git
+#git clone --recursive https://github.com/ufs-community/UFS_UTILS.git
```

Comment:
This will be changed after cron tests are successful?

Reply:
Yes. The only way I could test that script is by pointing to Denise's fork.